src/main/frontend/state.cljs

```diff
@@ -1631,10 +1501,6 @@
   [value]
```

Comment:
This fn is still used in frontend.handler/register-component-fns! but it is no longer used as evidenced by the removal below. Should we remove this unused state and fn?

Reply:
Yes, we should remove both this fn and the unused state.